web/client/components/map/openlayers/DrawSupport.jsx

```diff
@@ -514,6 +514,13 @@ class DrawSupport extends React.Component {
             geometry.setCoordinates(coordinates);
         }
 
+        if (this.props.drawMethod === "Polygon") {
+            /*
```

Comment:
You should remove this and add close polygon function to the WKT filter creation